test/e2e/framework/fixture.go

```diff
@@ -47,12 +47,34 @@ func NewKcpFixture(t *testing.T, cfgs ...KcpConfig) *KcpFixture {
 	artifactDir, dataDir, err := ScratchDirs(t)
 	require.NoError(t, err, "failed to create scratch dirs: %v", err)
 
+	f.Servers = map[string]RunningServer{}
+
+	if len(TestConfig.KubeConfig) > 0 {
```

Comment:
does it make sense to add `Disruptive: true` to the fixture config to opt-out of the possibly global fixture?

Reply:
I don't think we'll need it. For now, this is a developer-only feature intended to make it easier to get the tests to a hermetic state.  Long term, the goal is for all non-disruptive tests to move to shared fixture that they don't configure in any way, and for the disruptive tests to be the only tests defining test-local fixture.

Reply:
There is also a follow-up (#636) that switches to running test-managed syncer to enable a few more tests to run with shared fixture.